src/drivers/clock/raspi_systick.c

```diff
@@ -0,0 +1,108 @@
+/**
+ * @file
+ * @brief Core clocking device in Raspberry PI
+ *
+ * See BCM2835-ARM-Peripherals.pdf, 12 chapter for details.
+ *
+ * A frequency of the System Timer is 1 MHz.
+ *
+ * @date 02.07.15
+ * @author Vita Loginova
+ */
+
+#include <hal/clock.h>
+#include <hal/reg.h>
+#include <hal/system.h>
+#include <kernel/irq.h>
+#include <kernel/time/clock_source.h>
+
+#include <embox/unit.h>
+
+#define BCM2835_SYSTEM_TIMER_BASE 0x20003000
```

Comment:
Then I should rename all these modules a soon as the all part of RPi peripherals. Should I do it? It seems like there is no other board with bcm285 (at least popular, I couldn't find it).